_data/locales/signup.yml

```diff
@@ -23,14 +23,14 @@ privacy-notice:
   en: Privacy notice
   fr: Énoncé de confidentialité
 privacy-notice1:
-  en: Our newsletter feature uses <a href="https://mailchimp.com/">Mailchimp</a>, a third-party application. Sign-up is voluntary and subject to the <a href="https://mailchimp.com/legal/terms/">Terms of Use</a> for the Mailchimp platform.  When you sign up, your email address and language preference (English or French) is collected. No other personal information is collected.
-  fr: Notre fonction Bulletin de nouvelles fait appel &agrave; <a href="https://mailchimp.com/">Mailchimp</a>, une application tierce. L'adh&eacute;sion est libre et est assujettie aux <a href="https://mailchimp.com/legal/terms/">Conditions d'utilisation</a> associ&eacute;es &agrave; la plateforme Mailchimp. Au moment de l'adh&eacute;sion, le syst&egrave;me enregistre votre adresse &eacute;lectronique et votre langue de pr&eacute;f&eacute;rence (anglais ou fran&ccedil;ais). Aucun autre renseignement personnel n'est recueilli.
```

Comment:
Can you please change the Mailchimp.com link to the French version of the link which is: https://mailchimp.com/fr

Reply:
All fixed